lib/config.js

```diff
@@ -9,13 +9,13 @@ const remark = {
 
 const babel = {
   presets: [
-    'babel-preset-env',
-    'babel-preset-stage-0',
-    'babel-preset-react',
+    '@babel/preset-env',
+    '@babel/preset-react'
   ].map(require.resolve),
   plugins: [
     'babel-plugin-macros',
-    'babel-plugin-transform-runtime'
+    '@babel/plugin-transform-runtime',
+    '@babel/plugin-proposal-class-properties'
```

Comment:
I suspect the reason that you got the error about the import in the test is that there's still things missing from the plugins.

It's super confusing based on how the official babel upgrade doc is worded, but I think to get equivalent functionality we need _all_ of the stuff used in stage 0-2. I think all the extras where added by the babel upgarde tool to the `package.json` already. They probably just need to be added here.

And we _might_ need to add them to `mdx` loader as plugins as well? Some of those handle the syntax for `import`.

I could absolutely be wrong about all of this.